Dockerfile

```diff
@@ -65,31 +61,38 @@ RUN apt-get update && apt-get install -y --no-install-recommends  \
     vim                   \
     wget                  \
     zip                   \
-  && rm -rf /var/lib/apt/lists/* \
-  && apt-get clean all
+    libpq-dev           
 
 # Install conda.
 RUN cd /tmp && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
-    echo "${MINICONDA_SHA256} *Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh" | sha256sum -c - && \
-    /bin/bash Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
-    rm Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
-    echo "conda ${CONDA_VERSION}" >> $CONDA_DIR/conda-meta/pinned && \
-    conda config --system --prepend channels conda-forge && \
-    conda config --system --set auto_update_conda false && \
-    conda config --system --set show_channel_urls true && \
-    conda list python | grep '^python ' | tr -s ' ' | cut -d '.' -f 1,2 | sed 's/$/.*/' >> $CONDA_DIR/conda-meta/pinned && \
-    conda install --quiet --yes conda && \
-    conda install --quiet --yes pip && \
-    conda update --all --quiet --yes && \
+    export ARCH=`uname -m` && \
+    if [ "$ARCH" = "x86_64" ]; then \
+       echo "x86_64" && \
+       export MINICONDA_ARCH=x86_64 && \
+       export MINICONDA_SHA256=935d72deb16e42739d69644977290395561b7a6db059b316958d97939e9bdf3d; \
```

Comment:
Note that this export will only live inside this `RUN` command, which is different from before

Reply:
But it was in one `RUN` command, so the same?